aviary/core/aviary_group.py

```diff
@@ -54,12 +58,13 @@ class AviaryGroup(om.Group):
     def __init__(self, verbosity=None, **kwargs):
         super().__init__(**kwargs)
 
-        self.pre_mission = PreMissionGroup()
```

Comment:
I hadn't noticed before, but yeah, self.post_mission and self.pre_mission do collide with the subsystem names, which OpenMDAO later stores as attributes. Removing the local instantiation here, like you did, is probably cleaner for both.